docker-compose.yaml

```diff
@@ -37,10 +40,14 @@ services:
       - createbucket
       - temporalite
     environment:
-      EPP_SERVER_URI: http://api:3000
       TEMPORAL_SERVER: temporalite:7233
       AWS_ACCESS_KEY_ID: minio
       AWS_SECRET_ACCESS_KEY: miniotest
+      EPP_SERVER_URI: http://api:3000
+      BIORXIV_URI: http://nginx-mock:3002
+    links:
+      - nginx-mock:data-hub-api.elifesciences.org
+      - nginx-mock:api.biorxiv.org
```

Comment:
These changes add a lot of value to the developer experience. I want them in our toolkit but I don't want to disrupt the opportunity to hit datahub and biorxiv directly. Some of the changes in this docker-compose could be easily introduced in a separate docker-compose file. We are likely to want to, on a given run, target the production api's and then other times opt for this shorter feedback loop and use the mock services.

As I was involved with this work, I found the best way to do this review was to fire it up and then dig around. I have a feature branch where I adjusted some things to offer us this opportunity.

If we agree as a team that we want the mock services to be the default then we need to improve the README so it is clear how to set the appropriate environment variables to use the prod api's for datahub and biorxiv instead.

Since `BIORXIV_URL` is now a config option I don't believe the `nginx-mock:api.biorxiv.org` link is needed anyway.

I would prefer if we didn't introduce the link `nginx-mock:data-hub-api.elifesciences.org` but rather modify the mock docmaps to reference `http://nginx-mock/`.

Here is my feature branch. It expresses my view on how we could integrate with the mock services optionally plus some other stuff. Just intended to invite discussion about how we can begin getting value from this while also preserving a developer experience that integrates with the prod api's.

https://github.com/elifesciences/enhanced-preprints-import/compare/setup-dev-sandbox...setup-dev-sandbox-nlisgo?expand=1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5